recipes/deprecated/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "Deprecated" %}
+{% set version = "1.2.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "42b5dffc4023401148e9d2757f46a4a9fcccf114547a4620347ae45f9e77a21c" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --ignore-installed --no-deps .
+
+requirements:
+  host:
+    - python
+    - pip
+    - wrapt <2,>=1
```

Comment:
I checked upstream's `setup.py` and it looks like `wrapt` is only an install requirement. Do we really need it in the host section?

Reply:
let me remove that

Reply:
Seems one of the modules does use it, readding